tests/test_cv_output_format.py

```
@@ -0,0 +1,264 @@
+        assert sample_df["track_id"].dtype in (np.int64, np.int32, int)
+        assert (sample_df["track_id"] > 0).all(), "Non-positive track IDs"
+
+    def test_confidence_in_range(self, sample_df):
+        """confidence must be between 0 and 1."""
+        assert (sample_df["confidence"] >= 0).all(), "Confidence below 0"
+        assert (sample_df["confidence"] <= 1).all(), "Confidence above 1"
+
+    def test_team_is_string(self, sample_df):
+        """team should be string values."""
+        teams = sample_df["team"].unique()
+        valid_teams = {"home", "away", "referee", "unknown", "player"}
```

Ruff (0.15.2)

[error] Local variable `valid_teams` is assigned to but never used

Remove assignment to unused variable `valid_teams`

(F841)